{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "d7a6754b_53b205da",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1396442
      },
      "writtenOn": "2021-05-21T09:52:13Z",
      "side": 1,
      "message": "LGTM - not sure what\u0027s going on with that failure.",
      "revId": "fd4471e71950a1a0470f75d402c39e7ecdc81af1",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}